www/src/components/auth/AuthButton.tsx

```diff
@@ -0,0 +1,39 @@
+import { useAuth0 } from '@auth0/auth0-react';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+type BaseButtonProps = {
```

Comment:
Do we want to give unique names to Prop types or just name them `Props`? I think this is my coding style and I'm ok with either. Since they should be local to the file anyways they don't necessarily need a unique name aside from being verbose.

Reply:
I'm personally for keeping them unique just so I can see the type name in VSCode. I.e. 
![image](https://user-images.githubusercontent.com/43416725/120738165-9cdbc680-c4ac-11eb-924f-5f30b1adcc75.png)

Reply:
Fair fair, I'll follow suit with this style in my PR